app/Http/Requests/Three/PostRequest.php

```diff
@@ -29,7 +29,7 @@ public function rules()
             'northstar_id' => 'nullable|objectid',
             'why_participated' => 'nullable|string',
             'caption' => 'required|nullable|string|max:140',
-            'quantity' => 'required|nullable|integer',
```

Comment:
should this still be required but if there is no quantity, `null` should be sent? I tried to do this in the test (send through a `quantity` of `null` but it failed...

Reply:
I think setting `nullable` should allow you to send `null` as the quantity value. We should also allow quantity parameter to just be not sent at all and we don't receive a `quantity` param we should store it as `null` in the DB. 

From what I can tell ` 'nullable|integer` should be all we need. Was that not working in the tests?

Reply:
ah ok! Right now, the test sends `null` as the quantity and it works! 

Should I also write a test for not sending the quantity parameter at all? It's throwing a `500` for that but I can write a separate test to test both scenarios and fidget with it!

Reply:
Yea!

Reply:
if the quantity work becomes more than just creating the test- like application logic needs to be tweaked- we can do that in a separate ticket/PR to reduce the scope of this.

Reply:
ah i figured it out - I needed to use `isset` 🤦‍♂️